### PR TITLE
Updated File Integrity Operator release notes to include v0.1.24

### PR DESCRIPTION
**Pull request by sheriff-rh**
File Integrity Operator 0.1.24 release notes stub added.

Version(s):
4.6+

Link to docs preview:
[OpenShift File Integrity Operator 0.1.24](https://deploy-preview-45684--osdocs.netlify.app/openshift-enterprise/latest/security/file_integrity_operator/file-integrity-operator-release-notes.html#file-integrity-operator-release-notes-0-1-24)

### CODE REVIEW

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 1]
```diff
@@ -7,12 +7,30 @@ include::_attributes/common-attributes.adoc[]
...
+== OpenShift File Integrity Operator 0.1.24
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.24:
+
+* link:https://access.redhat.com/errata/RHBA-2022:1331[RHBA-2022:1331 OpenShift File Integrity Operator Bug Fix]
+
+[id="file-integrity-operator-0-1-24-new-features-and-enhancements"]
+=== New features and enhancements
+
+* You can now configure the maximum number of backups stored in the `FileIntegrity` Custom Resource (CR) with the `config.maxBackups` attribute. This allows AIDE database and log backups leftover from the `re-init` process to remain on a node. Older backups beyond the configured number are automatically pruned. The default is set to five backups.
```
Comment (mrogers950) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:26]: > This allows AIDE database and log backups leftover from the `re-init` process to remain on a node.

It's more about the amount of backups, so we should say something like:
This specifies the number of AIDE database and log backups leftover from the re-init process to keep on the node.

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 2]
```diff
@@ -7,12 +7,30 @@ include::_attributes/common-attributes.adoc[]
...
+
+[id="openshift-file-integrity-operator-0-1-24-bug-fixes"]
+=== Bug fixes
+* Previously, File Integrity Operator database corruption could occur when updating the File Integrity Operator from older versions. Now, upgrading to the latest version of the File Integrity Operator will complete successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049206[*BZ#2049206*])
```
Comment (mrogers950) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:30]: Upgrading the operator from pre-0.1.21 versions to 0.1.22 could cause the re-init feature to break. This was a result of the operator failing to update configMap resource labels. Now, upgrading to the latest version will fix the resource labels.

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 3]
```diff
@@ -7,12 +7,30 @@ include::_attributes/common-attributes.adoc[]
...
+
+* Previously, `worker-fileintegrity` pods remained in the `Initializing` phase due to `daemonSets` stuck in the `re-init` process. Now,`daemonSets` run to completion and the AIDE database `re-init` process executes successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2072058[*BZ#2072058*])
```
Comment (mrogers950) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:32]: Previously when enforcing the default configMap script contents, we compared the wrong data keys. This led to the aide-reinit script not being updated properly on an operator upgrade, and caused the re-init process to break.

**Review by tmalove — COMMENTED** [review]
/LGTM

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 4]
```diff
@@ -7,12 +7,30 @@ include::_attributes/common-attributes.adoc[]
...
+== OpenShift File Integrity Operator 0.1.24
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.24:
+
+* link:https://access.redhat.com/errata/RHBA-2022:1331[RHBA-2022:1331 OpenShift File Integrity Operator Bug Fix]
+
+[id="file-integrity-operator-0-1-24-new-features-and-enhancements"]
+=== New features and enhancements
+
+* You can now configure the maximum number of backups stored in the `FileIntegrity` Custom Resource (CR) with the `config.maxBackups` attribute. This specifies the number of AIDE database and log backups leftover from the `re-init` process to keep on the node. Older backups beyond the configured number are automatically pruned. The default is set to five backups.
```
Comment (jeana-redhat) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:26]: Does AIDE need a first-occurrence definition? Also:
```suggestion
* You can now configure the maximum number of backups stored in the `FileIntegrity` Custom Resource (CR) with the `config.maxBackups` attribute. This attribute specifies the number of AIDE database and log backups left over from the `re-init` process to keep on the node. Older backups beyond the configured number are automatically pruned. The default is set to five backups.
```

File: security/file_integrity_operator/file-integrity-operator-release-notes.adoc [hunk 5]
```diff
@@ -7,12 +7,30 @@ include::_attributes/common-attributes.adoc[]
...
+
+[id="openshift-file-integrity-operator-0-1-24-bug-fixes"]
+=== Bug fixes
+* Upgrading the Operator from versions older than 0.1.21 to 0.1.22 could cause the `re-init` feature to fail. This was a result of the Operator failing to update `configMap` resource labels. Now, upgrading to the latest version will fix the resource labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049206[*BZ#2049206*])
```
Comment (jeana-redhat) [inline security/file_integrity_operator/file-integrity-operator-release-notes.adoc:30]:
```suggestion
* Previously, upgrading the Operator from versions older than 0.1.21 to 0.1.22 could cause the `re-init` feature to fail. This was a result of the Operator failing to update `configMap` resource labels. Now, upgrading to the latest version fixes the resource labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049206[*BZ#2049206*])
```

**Review by jeana-redhat — COMMENTED** [review]
Couple super minor nit-picks for ya :) Otherwise LGTM

**Pull request merged**